src/parser.rs

```diff
@@ -23,7 +23,7 @@ pub struct Parser {}
 type ParseResult<T> = Result<(T, Vec<ParseErrorDetails>), ParseErrorDetails>;
 
 impl Parser {
-    pub const REST_FILE_EXTENSIONS: [&str; 2] = ["http", "rest"];
+    pub const REST_FILE_EXTENSIONS: [&'static str; 2] = ["http", "rest"];
```

Comment:
As a heads up this changed is inspired by the compiler output.
```bash
➜ cargo check --all --all-features
warning: `&` without an explicit lifetime name cannot be used here
  --> src/parser.rs:26:38
   |
26 |     pub const REST_FILE_EXTENSIONS: [&str; 2] = ["http", "rest"];
   |                                      ^
   |
   = warning: this was previously accepted by the compiler but is being phased out; it will become a hard error in a future release!
   = note: for more information, see issue #115010 <https://github.com/rust-lang/rust/issues/115010>
   = note: `#[warn(elided_lifetimes_in_associated_constant)]` on by default
help: use the `'static` lifetime
   |
26 |     pub const REST_FILE_EXTENSIONS: [&'static str; 2] = ["http", "rest"];
   |
```